OLE-objects/Get-file-path-of-linked-OLE-Object/.NET/Get-file-path-of-linked-OLE-Object/Program.cs

```diff
@@ -0,0 +1,17 @@
+﻿using Syncfusion.Presentation;
+
+//Load or open an PowerPoint Presentation
+using FileStream inputStream = new(Path.GetFullPath(@"../../../Data/Template.pptx"), FileMode.Open, FileAccess.Read, FileShare.ReadWrite);
+//Open the specified presentation.
+using IPresentation pptxDoc = Presentation.Open(inputStream);
+//Gets the first slide of the Presentation
+ISlide slide = pptxDoc.Slides[0];
+//Gets the Ole Object of the slide
+IOleObject oleObject = slide.Shapes[1] as IOleObject;

```

Comment:
@DharanitharanA  - Please let me know the purpose of `Get-file-path-of-linked-OLE-Object/Data/HelloWorld.docx` document?

Reply:
We will get the linked OLE object file path from this sample. This HelloWorld.docx document is used as the OLE object.